etc/sample-input/wav-wrap.json

```diff
@@ -0,0 +1,39 @@
+{
+  "Job": {
+    "Id": "porter-sample-wav-wrap",
+    "Source": {
+      "Mode": "HTTP",
+      "URL": "https://prx-tech.s3.us-east-1.amazonaws.com/example/test.mp2"
+    },
+    "Tasks": [
+      {
+        "Type": "WavWrap",
+        "Destination": {
+          "Mode": "AWS/S3",
+          "BucketName": "prx-porter-sandbox",
+          "ObjectKey": "dev/sample/wav-wrap/10000.wav",
+          "ContentType": "REPLACE",
+          "Parameters": {
+            "ContentDisposition": "attachment",
+            "Metadata": {
+              "MyMetadataKey": "MyMetadataValue"
+            }
+          }
+        },
+        "Chunks": {
+          "cart": {
```

Comment:
Ideally all job parameters would be PascalCase, for consistency with the the other task types. It seems like there's not too many metadata keys, even across all the chunk types (that I could find), so I like the idea of the Porter job definition properties being decoupled from the underlying library message format.

Reply:
My thought was that "Chunks" was the parameter, and the object inside it is all data.

Reply:
Ok, looks like I can consistently convert PascalCase to CamelCase, so I did that.
Converted the `Chunks` to an array of chunks instead of a Map to make that easier.